src/components/GeneratorHeader/GeneratorHeader.module.scss

```diff
@@ -0,0 +1,41 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap'; // temp?
+
+.root {
+    box-sizing: border-box;
+    background-color: #000;
+    font-family: Roboto, sans-serif;
+    font-weight: 700;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+
+    & a {
```

Comment:
I suggest creating a new class for it. Large style files using this approach are hard to maintain.